app/access/app_event_access.rb

```diff
@@ -31,8 +31,7 @@ def delete?(_object)
     end
 
     def index?(_object_class, _params=nil)
```

Comment:
In general, is it necessary to fill in this logic in both `index?` and `index_with_token?` for all these access classes? From looking at https://github.com/cloudfoundry/cloud_controller_ng/blob/7b1848e6b3d33ecc486adb55fbbe35fe44fb2cb3/app/controllers/base/model_controller.rb#L239-L251 the `index_with_token?` is called first and the request will never reach `index?`. Also the `index_with_token?` error seems more appropriate for this case (though, we aren't very consistent with which access method we use, in general).

Reply:
Broadly speaking, it seems to me both safer, and moreso, consistant for us to double up on both. 

It doesn't really cost anything for us to have the same thing both places, and even before the change, the same comment about how "filtering happens later" in the old code was often in both places anyways, so I just went with both.

Reply:
My main concern is adding code that isn't necessary to achieve the desired behavior and may never execute.

I think the "original sin" was https://github.com/cloudfoundry/cloud_controller_ng/commit/1d6b036fbf4c5c84e092f12b7d3d62996a5a9fb8, which inlined all these methods (not disparaging; it made sense at the time). These methods made sense as base class defaults, but not necessarily included in each access class.

That said, especially since this is v2 only, it's also reasonable to just continue the current pattern, messy as it is. I'm not going to block on this.